index.html

```diff
@@ -756,7 +756,7 @@ <h3>
           {{RTCIceTransport/icecandidatepairnominate}} at |transport|, using {{RTCIceCandidatePairEvent}}, with the
           {{Event/cancelable}} attribute initialized to <code>true</code>, and the {{RTCIceCandidatePairEvent/local}} and
           {{RTCIceCandidatePairEvent/remote}} attributes initialized to the local and remote candidates, respectively, of
-          |candidatePair|.
+          <var>candidatePair</var>.
```

Comment:
What's wrong with the |candidatePair| syntax? Is it not working? It appears used elsewhere, e.g. with |accepted|. Please change back before merging unless there was a reason.

Reply:
Reverted.

I was trying to see if it fixed the highlighting of identically-named variables in adjecent algorithms. Eg. `candidatePair` in "[nominate a candidate pair](https://w3c.github.io/webrtc-extensions/#rtcicetransport-nominate)" and "[remove a candidate pair](https://w3c.github.io/webrtc-extensions/#rtcicetransport-remove)", or `sender` in [addTrack](https://w3c.github.io/webrtc-pc/#dom-rtcpeerconnection-addtrack) and [removeTrack](https://w3c.github.io/webrtc-pc/#dom-rtcpeerconnection-removetrack). It does not.